stable/kubernetes-dashboard/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{template "fullname" .}}
+  labels:
+    app: {{ template "fullname" . }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    kubernetes.io/cluster-service: "true"
+    release: {{.Release.Name | quote }}
+  namespace: {{ default "kube-system" .Values.namespace }}
```

Comment:
Rule of thumb with configurable values: if it's configurable, set the default value in `values.yaml` and change the reference in the resource to that, e.g. `namespace: {{ .Values.namespace }}`

At the moment, the deployment installs into kube-system by default but the release thinks it's "default":

```
NOTES:
*********************************************************************************
*** PLEASE BE PATIENT: kubernetes-dashboard may take a few minutes to install ***
*********************************************************************************

Get the Kubernetes Dashboard URL by running:

  export POD_NAME=$(kubectl get pods --namespace default -l "app=yucky-dog-kubernetes-das" -o jsonpath="{.items[0].metadata.name}")
  echo http://127.0.0.1:8080/
  kubectl port-forward $POD_NAME 8080:80

$ export POD_NAME=$(kubectl get pods --namespace default -l "app=yucky-dog-kubernetes-das" -o jsonpath="{.items[0].metadata.name}")
No resources found.
error: error executing jsonpath "{.items[0].metadata.name}": array index out of bounds: index 0, length 0
```